server/src/test/java/org/elasticsearch/index/engine/LuceneChangesSnapshotTests.java

```diff
@@ -178,9 +187,10 @@ public void testUpdateAndReadChangesConcurrently() throws Exception {
         }
         boolean onPrimary = randomBoolean();
         List<Engine.Operation> operations = new ArrayList<>();
-        int numOps = scaledRandomIntBetween(1, 1000);
+        int numOps = frequently() ? scaledRandomIntBetween(1, 1000) : scaledRandomIntBetween(5000, 20_000);
```

Comment:
Maybe we can add a bit higher chance of going above 1000 here?
```suggestion
        int numOps = frequently() ? scaledRandomIntBetween(1, 1500) : scaledRandomIntBetween(5000, 20_000);
```